src/api/app/assets/javascripts/webui2/autocomplete.js

```diff
@@ -5,10 +5,19 @@ $(document).ready(function() {
       appendTo:  $(this).data('append'),
       source:    $(this).data('source'),
       minLength: 2,
+      create: function() {
+        // disable submit button
+        $('input.btn.btn-sm.btn-primary.px-4').prop("disabled", true);
+      },
       search: function() {
         $(this).prev().find('i').toggleClass('fa-search fa-spinner fa-spin');
       },
-      response: function() {
+      response: function(event, ui) {
+        if(ui.content.length > 0) {
```

Comment:
It's still possible to submit the form without an existing user. In the review app, we have the users `user_1` and `user_2`, so typing `u` in the autocomplete will return these 2 users and enable the submit button. Clicking on the submit button will take us to the `add_person` view.

Instead of checking if some users are returned, we need to check if one of the returned users is an exact match. If it is, then enable the submit button. If not, leave it disabled.